### PR TITLE
Major copy changes to native.mdx

### PR DESCRIPTION
**Pull request by MindyCoW**
Rewrote a bunch of the page to simplify language & correct minor grammar errors

# Description

- Added more context regarding the pros and cons of wrapping native tokens vs. using the Eth-flow
- Fleshed out the steps for using the Eth-flow
- Simplified language where possible
- Corrected grammar errors

# Changes

- [ ] ...
- [ ] ...

### CODE REVIEW

**Comment by fedgiac** [comment]
Something I'd like to see in this page is that ETH can indeed be bought with CoW Swap without any issue. It's just selling that doesn't work well. Do you think you would be able to add it nicely or is it too much information?
Otherwise, I like the changes.

**Comment by MindyCoW** [comment]
Good idea! Made the change. 

EDIT: Sorry Federico =( in the update title I attributed this suggestion to mfw =((

**Comment by mfw78** [comment]
No credit taken by me for @fedgiac's great work :smile:

File: docs/cow-protocol/tutorials/cow-swap/native.mdx [hunk 1]
```diff
@@ -6,38 +6,43 @@ import EthFlowTip from '../../../partials/_eth_flow.mdx'
...
-Their "wrapped" token equivalents, e.g. Wrapped Ether (WETH) & Wrapped xDAI (wxDAI), are [ERC20-compatible](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/) versions of native tokens. 
-They're created to improve interoperability with decentralized applications and always allow to "wrap" or "un-wrap" into one another at a rate of 1:1.
+"Wrapped" tokens - for example, Wrapped Ether (WETH) & Wrapped xDAI (wxDAI) - are [ERC20-compatible](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/) versions of these native tokens. 
+They're created to improve interoperability with decentralized applications. The native token and its wrapped version are equivalent to each other, and a user is always allowed to turn one into the other ("wrap" or "unwrap") at a rate of 1:1.
 
-CoW Protocol is only capable of handling ERC20 tokens.
-This is because its intent-based architecture relies on solvers, or more precisely the vault relayer, to pull in the sell token on behalf of its traders.
+_Buying_ native tokens such as ETH on CoW Protocol is straightforward. However, CoW Protocol is only natively capable of _selling_ ERC20 tokens - not native tokens.
```
Comment (mfw78) [inline docs/cow-protocol/tutorials/cow-swap/native.mdx:14]:
```suggestion
_Buying_ native tokens such as ETH on CoW Protocol is straightforward. However, CoW Protocol is only capable of _selling_ ERC20 tokens - not native tokens. 
```

File: docs/cow-protocol/tutorials/cow-swap/native.mdx [hunk 2]
```diff
@@ -6,38 +6,43 @@ import EthFlowTip from '../../../partials/_eth_flow.mdx'
...
+
+After wrapping the token, you can simply select the wrapped token as your sell token and continue with your swap.
+
+
 ## Eth-flow
 
-In an attempt to smooth the user experience, CoW Protocol has introduced the [Eth-flow](/cow-protocol/reference/contracts/periphery/eth-flow) contract. This allows users to automate the [above process](#wrap-native-tokens), all in one **on-chain** transaction.
+In an attempt to smooth the user experience, CoW Protocol has introduced the [Eth-flow](/cow-protocol/reference/contracts/periphery/eth-flow) contract. This allows users to automate the [above process](#wrap-native-tokens) and wrap + swap in a single **on-chain** transaction. 
+
+Using the Eth-flow is faster than wrapping, then swapping. However, because an Eth-flow transactions is on-chain (unlike most CoW Swap swaps), the user must pay an up-front gas fee in the native token. This fee is not refunded if the order fails.
```
Comment (mfw78) [inline docs/cow-protocol/tutorials/cow-swap/native.mdx:43]: I'm not sure about the assertion of "faster" in this context?

**Review by mfw78 — APPROVED** [review]
Some minor points, will merge and then follow-up on discussion.

**Pull request merged**